src/layers/models/fileValidator.ts

```diff
@@ -103,6 +98,39 @@ export class FileValidator {
     );
   }
 
+  public async validateInfoData(files: string[], originDirectory: string): Promise<void> {
+    await Promise.all(
```

Comment:
add try catch to this method and in the catch throw message "failed to validate File GDAL Info"